test/MOF/jump_integration.jl

```diff
@@ -0,0 +1,46 @@
+function MOI.copy_to(dest, src::Model; kwargs...)
+    if src.nlp_data !== nothing
+        MOI.set(src, MOI.NLPBlock(), JuMP._create_nlp_block_data(src))
+    end
+    MOI.copy_to(dest, backend(src); kwargs...)
+end
+
+function MOI.copy_to(dest::Model, src; kwargs...)
+    MOI.copy_to(backend(dest), src; kwargs...)
+    nlp_block = MOI.get(src, MOI.NLPBlock())
+    if nlp_block !== nothing
+        MOI.set(dest, MOI.NLPBlock(), nlp_block)
+    end
+    return
+end
+
+using JuMP, MathOptFormat, Ipopt
+
+jump_1 = Model()
+@variable(jump_1, x, start = 1)
+@variable(jump_1, y, start = 2.12)
+@variable(jump_1, z >= 1)
+@NLobjective(jump_1, Min, x * exp(x) + cos(y) + z^3 - z^2)
+
+mof_1 = MathOptFormat.MOF.Model()
+MOI.copy_to(mof_1, jump_1)
+MOI.write_to_file(mof_1, "mof.json")
+
+mof_2 = MathOptFormat.MOF.Model()
+MOI.read_from_file(mof_2, "mof.json")
+
+jump_2 = Model(with_optimizer(Ipopt.Optimizer))
+MOI.copy_to(jump_2, mof_2)
```

Comment:
@mlubin, demo round-tripping a JuMP model through MOF and solving it via Ipopt.

Reply:
FWIW it's better to test the two parts separately (JuMP to MOF and MOF to JuMP) instead of testing the round trip.